NEWS.md

```diff
@@ -1,5 +1,9 @@
 # pkgload (development version)
 
+* The `reset` argment of `load_all()` is no longer supported because preserving
+  the namespace requires unlocking its environment, which is no longer possible
+  in recent versions of R.
```

Comment:
I think you could add something like: It's also rarely used since the performance benefits went away some time ago.

Reply:
It now says:  It should no longer be necessary as the performance issues caused by resetting the namespace were resolved a while ago.